dist/performance.js

```diff
@@ -84,6 +85,100 @@ function getWebVitalsJS() {
     }).map(har => har.url);
 }
 
+function testPropertyStringInResponseBodies(regex) {
+    try {
+        return script_response_bodies.some(body => {
+            if (!body.response_body) {
+                return false;
+            }
+            return regex.test(body.response_body);
+        });
+    } catch (error) {
+        return false;
+    }
+}
+
+function getGamingMetrics(rawDoc) {
+    let returnObj = {};
+    const regexForCheckChromeLH = new RegExp(/.{1}userAgent.{1,100}(?:Chrome-Lighthouse|Google Lighthouse).{2}/)
+    const regexForCheckGTmetrix = new RegExp(/.{1}userAgent.{1,100}(?:GTmetrix|gtmetrix.com).{2}/)
+    const regexForCheckPageSpeed = new RegExp(/.{1}userAgent.{1,100}(?:PageSpeed).{2}/)
+
+    //inline scripts check
+    Array.from(rawDoc.querySelectorAll('script:not([src])')).forEach(script => {
+        let scriptTagCode = script.innerHTML;
+        if (regexForCheckChromeLH.test(scriptTagCode)){
+            returnObj['detectUA-ChromeLH'] = true;
+        }
+        if(regexForCheckGTmetrix.test(scriptTagCode)){
+            returnObj['detectUA-GTmetrix'] = true;
+        }
+        if(regexForCheckPageSpeed.test(scriptTagCode)){
+            returnObj['detectUA-PageSpeed'] = true;
+        }
+    });
+
+    //check external scripts response bodies
+    if (testPropertyStringInResponseBodies(regexForCheckChromeLH)) {
+        returnObj['detectUA-ChromeLH'] = true;
+    }
+
+    if (testPropertyStringInResponseBodies(regexForCheckGTmetrix)) {
+        returnObj['detectUA-GTmetrix'] = true;
+    }
+
+    if (testPropertyStringInResponseBodies(regexForCheckPageSpeed)) {
+        returnObj['detectUA-PageSpeed'] = true;
+    }
+
+    //https://www.debugbear.com/blog/optimizing-web-vitals-without-improving-performance
+    //catch lcp animation / cls animation & overlay hack
+    const regexForCheckfadeInAnimation = new RegExp(/this.style.animation.{1,10}.fadein.{1,20}.forwards/)
+    Array.from(rawDoc.querySelectorAll('img')).forEach(el => {
+        let onloadVal = el.getAttribute('onload');
+        if (onloadVal !== null) {
+            if(regexForCheckfadeInAnimation.test(onloadVal)) {
+                returnObj['lcpAnimation'] = true;
+            }
+        }
+        let styleObj = el.style;
+        if (styleObj['pointer-events'] == 'none' &&
+                styleObj['position'] == 'absolute' &&
+                styleObj['width'] == '99vw' &&
+                styleObj['height'] == '99vh') {
+            returnObj['lcpOverlay'] = true;
+        }
+    });
+
+    //add logic for svg & body overlay
+    Array.from(rawDoc.querySelectorAll('svg')).forEach(svg => {
```

Comment:
Good news - [the hack](https://www.devisedlabs.com/blog/largest-contentful-paint-lcp-hack) for this is actually an svg in the source of an image tag, so we can omit the `svg` query.